src/improvements/template/TransferOwners.sol

```diff
@@ -0,0 +1,181 @@
+        return "ProxyAdminOwner";
+    }
+
+    /// @notice Returns the storage write permissions required for this task.
+    function _taskStorageWrites() internal pure virtual override returns (string[] memory) {
+        string[] memory storageWrites = new string[](4);
+        storageWrites[0] = "DisputeGameFactoryProxy";
+        storageWrites[1] = "ProxyAdmin";
+        storageWrites[2] = "PermissionedWETH";
+        storageWrites[3] = "PermissionlessWETH";
+        return storageWrites;
+    }
+
+    /// @notice Sets up the template with the new owner from a TOML file.
+    function _templateSetup(string memory _taskConfigFilePath) internal override {
+        super._templateSetup(_taskConfigFilePath);
+        string memory toml = vm.readFile(_taskConfigFilePath);
+        newOwner = abi.decode(vm.parseToml(toml, ".newOwner"), (address));
+
+        // Only allow one chain to be modified at a time with this template.
+        SuperchainAddressRegistry.ChainInfo[] memory _parsedChains = superchainAddrRegistry.getChains();
+        require(_parsedChains.length == 1, "Must specify exactly one chain id to transfer ownership for");
+        activeChainInfo = _parsedChains[0]; // Store the ChainInfo struct
+
+        // The discovered SuperchainConfig address must match the SuperchainConfig address in the standard config.
+        address superchainConfig = superchainAddrRegistry.getAddress("SuperchainConfig", activeChainInfo.chainId);
+
+        // Discover OP Mainnet and OP Sepolia chains. We do this to get access to the latest SuperchainConfig addresses.
+        // We assume that these chains are always using the standard config.
+        _validateSuperchainConfig(superchainConfig);
+    }
+
+    /// @notice Builds the actions for transferring ownership of the DisputeGameFactory, DWETH contracts and ProxyAdmin.
+    function _build() internal override {
+        ProxyAdmin proxyAdmin = ProxyAdmin(superchainAddrRegistry.getAddress("ProxyAdmin", activeChainInfo.chainId));
+        IDisputeGameFactory disputeGameFactory =
+            IDisputeGameFactory(superchainAddrRegistry.getAddress("DisputeGameFactoryProxy", activeChainInfo.chainId));
+        IDelayedWETH permissionedWETH = _getDWETH("PermissionedWETH", activeChainInfo.chainId);
+        IDelayedWETH permissionlessWETH = _getDWETH("PermissionlessWETH", activeChainInfo.chainId);
+
+        // Transfer ownership of the DisputeGameFactory to the new owner.
+        performOwnershipTransfer(address(disputeGameFactory), newOwner);
+
+        // Check if PermissionedWETH exists and is ownable. If it is, transfer ownership to the new owner.
+        if (_isDWETHOwnable(permissionedWETH) && address(permissionedWETH) != address(0)) {
+            performOwnershipTransfer(address(permissionedWETH), newOwner);
+        } else {
+            console.log("PermissionedWETH not found on chain %s, not performing transfer", activeChainInfo.chainId);
```

Comment:
nit, out of date comment
```suggestion
            console.log("PermissionedWETH not found or not ownable on chain %s, not performing transfer", activeChainInfo.chainId);
```

Reply:
done: https://github.com/ethereum-optimism/superchain-ops/pull/972/commits/7cf7034df87b699639fa18fb0fe8509ca64ee264